webapp/sources/rudder/rudder-web/src/test/resources/configuration-repository/category.xml

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
there is only one remaining interesting file below (`TestMigrateSystemTechnique7_0.scala`), other are just technique files for the test techniqueRepo